engine/Shopware/Controllers/Backend/Config.php

```diff
@@ -287,6 +287,18 @@ public function getListAction()
             $data = $query->getArrayResult();
         }
 
+        if ($name === 'document') {
+            // Translate documents
```

Comment:
Doesn't actually translate documents, maybe "Translates document type names" is more precise?